### ISSUE TITLE
Create a comprehensive metadata record for ASGS data

### ISSUE THREAD

**Issue by dr-shorthair** [CSIRO-enviro-informatics/asgs-dataset#8]
https://github.com/CSIRO-enviro-informatics/asgs-dataset/blob/master/asgs_dataset/view/dcat.ttl was created as a basic metadata record. 
It should be 
(a) updated to match DCAT-2 best practice
(b) serialised in JSON
(c) embedded in the asgs-dataset and landing page header 

Supersedes #3

**Comment by nicholascar** [comment]
@dr-shorthair note that the pyLDAPI can serve up conneg for the landing page so while it's a great idea to embed the DCAT2 record as JSON-LD in the HTML landing page, please can the same content in RDF be available at `curl -H 'Accept: text/turtle' http://linked.data.gov.au/dataset/asgs2016` etc.? This makes AGLDWG cataloguing even easier than parsing the JSON-LD and is the correct Linked Data thing to do.

The DCAT  content can all be taken from 1 file in pyLDAPI, for the JSON-LD-in-HTML and RDF delivery formats.

**Comment by ashleysommer** [comment]
@nicholascar 
Yes, I had already planned to make the same metadata available as both embedded json-ld _and_ in the RDF representation of the dataset.

**Comment by dr-shorthair** [comment]
OK - I see that request is currently getting the 'DCAT' record that you created. 

Please inspect and critique the updated version I've now made - https://github.com/CSIRO-enviro-informatics/asgs-dataset/blob/master/asgs_dataset/view/dcat.ttl 

I distinguished three resources in this record:
1. the original ASGS dataset, packaged by ABS Geospatial Solutions and delivered as a Geopackage - http://data.gov.au/dataset/asgs-2016-edition-boundaries
2. the linked-data version of the dataset, structured according to asgs-ont - http://linked.data.gov.au/dataset/asgs2016 
3. the service endpoint at https://asgsld.net/2016/ 

There is a question as to whether delivering the _metadata record_ would meet expectations, when de-referencing the _dataset URI._ Of course this is the issue addressed by conneg-by-profile, but should probably also be reflected in the 'alternates' view.

**Comment by nicholascar** [comment]
> should probably also be reflected in the 'alternates' view.

I'll check over all the 'alternates' views shortly since I'll be looking for evidence of implementation for Conneg by Profile & PROF. I'll push a change to pyLDAPI that will do most of the work but will work with @ashleysommer and others to ensure the change is non-breaking for all the LocI implementations.
